### DISCUSSION TITLE
Can't get password reset link when request for Reset your password

### DISCUSSION THREAD

**Discussion by TrunkOnBonk** [community/community#118892] [New to GitHub]
I was trying to retrieve the password of an old account. But when I sent a forgotten password request, even though github still said it sent an email to my account, when I checked my mailbox, I didn't see any mail from github. What should I do now?

**Comment by monwarqx1250** [comment]
did you checked your spam mail? sometimes website like github and facebook stored on the spam mail box.

**Reply by santoshcandy** [reply]
same problem

**Comment by dishb** [comment]
I have the exact same issue. Requested a password reset link and got it around 2 hours later. Was asleep and the link expired. Tried again just now and the link has not arrived in my email. Yes, I checked spam.

**Comment by 9001** [comment]
I am likely hitting the same issue; currently unable to log into an alt account because github claims to have sent a confirmation code to the associated email, but it's not arriving in my inbox.

The account in question is entirely non-critical, so it definitely did not need 2FA protection, but unfortunately I forgot to disable it.

**Comment by piro92** [comment]
Same here

**Comment by walidmh19** [comment]
I got an email that says "We recently noticed that your GitHub account had a suspicious login. Out of an abundance of caution, we made the decision to force a password reset for the account associated with this email address.

We have no reason to believe that this was the result of a GitHub vulnerability. While we can’t speak to your particular case, in other similar account takeovers our users have told us that infostealer malware was discovered on their system.

Follow the link below to request a password reset token.

https://github.com/password_reset"

when I try to reset the password it says that my email is not associated to any account.

**Comment by Campajolaura** [comment]
Same problem. Password not valid, send new password, but link never arrived. No spam messages. So frustrating.

**Reply by st396gt** [reply]
Same here.  I created this throwaway account at Hotmail because I couldn't get past the "verification" barrier to leave a message.   
Has anyone been successful at getthing this fixed?

**Comment by ViniciusMarins** [comment]
Did you checked your spam mail? sometimes github and facebook stored on the spam mail box, anyway maybe request again

**Comment by QKQS** [comment]
git账户用户名和密码给忘了，百度说在GitHub上面能够重置，应该怎么样进行重置密码？我到现在为止还没有找到办法进行git仓库的代码使用，请求大佬给点意见和建议。
